test/rocwmma_options.hpp

```diff
@@ -69,6 +78,31 @@ namespace rocwmma
                 mOmitCout = true;
         }
 
+        bool setEmulationOption(std::string const& value)
```

Comment:
parsing of strings should probably be in the parsing body. This function should only care about the EmulationOption enum value